collector/src/bin/rustc-perf-collector/main.rs

```diff
@@ -255,35 +345,27 @@ fn main_result() -> Result<i32, Error> {
             let out_repo = get_out_repo(false)?;
             let sysroot = Sysroot::install(&commit, "x86_64-unknown-linux-gnu", false, false)
                 .map_err(SyncFailure::new)?;
+            let build_kinds = vec![BuildKind::Check, BuildKind::Debug, BuildKind::Opt];
```

Comment:
Can we make this a constructor call too? Should be able to return `&[]` I think without the `vec!`.